src/res_scanner.ml

```diff
@@ -350,26 +350,32 @@ let scanStringEscapeSequence ~startPos scanner =
     ()
 
 let scanString scanner =
-  let offs = scanner.offset in
+  (* assumption: we've just matched a quote *)
```

Comment:
More dangerous here but it's ok. Only used in this file in one place.

Reply:
Yes, this is an unofficial implicit assumption I regularly try to follow.